pkg/cluster/k8sres.go

```diff
@@ -2331,6 +2367,7 @@ func (c *Cluster) generateLogicalBackupJob() (*batchv1.CronJob, error) {
 		false,
 		"",
 		false,
+		[]v1.TopologySpreadConstraint{},
```

Comment:
atm we reuse configured tolerations also for logical backup so I guess we can do the same with constraints

Reply:
Ok I got it.